src/common/snippets/src/lowered/linear_ir.cpp

```diff
@@ -379,6 +383,14 @@ VectorDims LinearIR::get_master_shape() const {
     return master_shape;
 }
 
+void LinearIR::init_runtime_configurator() {
+    m_lowered_config = m_runtime_configurator->init(*this);
+}
+
+const RuntimeConfig& LinearIR::configure_runtime_args() {
+    return m_runtime_configurator->update(*this);
```

Comment:
Also discussed also in op::Subgraph: not sure why we need these bypass methods in LIR interface.
Look at the signatures of configurator's `init()` and `update()` methods: it takes LIR as an argument, and this makes perfect sense, since the configurator needs an IR to compute runtime properties. But the opposite is not valid, LIR doesn't need the configurator at all, it doesn't affect it's ability to maintain the graph state and apply transformations.